GitHubPullRequests.ps1

```diff
@@ -422,6 +416,13 @@ filter New-GitHubPullRequest
         $acceptHeader = 'application/vnd.github.shadow-cat-preview+json'
     }
 
+    if (-not $PSCmdlet.ShouldProcess($Title, 'Create GitHub Pull Request'))
```

Comment:
Not guaranteed to have a `Title`.  The user might be creating a pull request from an `Issue`.

Reply:
Code refactored.